### PR TITLE
Sso sim

### PR DESCRIPTION
**Pull request by keskitalo**
This branch was developed with @aamirmali and @Zhilei. It adds Solar System Object (SSO) simulation capability to `toast_so_sim.py`.

### CODE REVIEW

**Comment by tskisner** [comment]
This branch needs a major rebase against master.  Looking at the network graph, it has diverged quite a bit.

**Comment by keskitalo** [comment]
@tskisner Indeed, I would like to merge this branch and start anew with less convoluted branch history. Since this branch is functional (if not feature-complete) it seems like a good time to merge.

File: pipelines/get_wafer_offset.py
Comment (tskisner) [inline pipelines/get_wafer_offset.py:81]: Note that I do a duplicate calculation in the visualization tools (https://github.com/simonsobs/sotodlib/blob/f560e3196c36ebdd890516e58bac94da24f7708d/sotodlib/vis_hardware.py#L108).  We should make a function that does this and can be used in both places.  That can wait until after this is merged, but let's not forget about it.

File: sotodlib/observe/sim_sso.py
Comment (tskisner) [inline sotodlib/observe/sim_sso.py:50]: It looks like this file is duplicate code that was split up into the other correct source files.  If so, can we delete it from this branch before merging?  Note that the `observe` directory does not even exist in the sotodlib package.

**Review by tskisner — CHANGES_REQUESTED** [review]
Looks fine after deleting that stale file (assuming everything in there has been moved to other sources)

File: pipelines/get_wafer_offset.py
Comment (keskitalo) [reply pipelines/get_wafer_offset.py:81]: Agreed. We don't want duplicated code. The context for this pipeline is that we needed to automate the offset estimation for shell scripting.

File: sotodlib/observe/sim_sso.py
Comment (keskitalo) [reply sotodlib/observe/sim_sso.py:50]: Indeed! Thanks for catching this.

**Review by tskisner — APPROVED** [review]
Thanks for the fixes.  Fine to merge after resolving the conflicts.

**Pull request merged**
